src/components/map/Map.scss

```diff
@@ -0,0 +1,24 @@
+// *{
+//     border: 2px solid red;
+// }
+
+main{
```

Comment:
Don't style `main` directly, other people are using this tag! Only _very, very rarely_ is it okay to style HTML elements directly like this. Give it a class of some sort.

In fact, what role is the `position: relative` playing?